### PR TITLE
Solved possible CMake variable shadowing issue

### PR DESCRIPTION
**Pull request by MiguelBarro**

### CODE REVIEW

File: cmake/common/gtest.cmake [hunk 1]
```diff
@@ -120,6 +120,7 @@ macro(add_gtest)
                     COMMAND ${command} --gtest_filter=${GTEST_GROUP_NAME}.${GTEST_TEST_NAME}:*/${GTEST_GROUP_NAME}.${GTEST_TEST_NAME}/*)
 
                 # Add environment
+                set(GTEST_ENVIRONMENT "")
```
Comment (richiware) [inline cmake/common/gtest.cmake:123]: Always `unset` the variable.

File: cmake/common/gtest.cmake [hunk 2]
```diff
@@ -143,6 +144,7 @@ macro(add_gtest)
         add_test(NAME ${test} COMMAND ${command})
 
         # Add environment
+        set(GTEST_ENVIRONMENT "")
```
Comment (richiware) [inline cmake/common/gtest.cmake:147]: Always `unset` the variable

**Review by richiware — CHANGES_REQUESTED** [review]

File: cmake/common/gtest.cmake (hunk 1 above)
Comment (MiguelBarro) [reply cmake/common/gtest.cmake:123]: 👍

File: cmake/common/gtest.cmake (hunk 2 above)
Comment (MiguelBarro) [reply cmake/common/gtest.cmake:147]: 👍

**Review by richiware — APPROVED** [review]

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/3011/)
* Linux aarch64Build Status
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/2698/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/3022/)

**Pull request merged**